flutter/lib/src/widgets_binding_observer.dart

```diff
@@ -20,11 +24,31 @@ class SentryWidgetsBindingObserver with WidgetsBindingObserver {
     Hub? hub,
     required SentryFlutterOptions options,
   })  : _hub = hub ?? HubAdapter(),
-        _options = options;
+        _options = options,
+        _screenSizeStreamController = StreamController(sync: true) {
```

Comment:
Maybe use the `SynchronousStreamController` directly then or better the `StreamController` interface?

Reply:
@marandaneto it's a matter of taste, both will work. I prefer to leave StreamController interface, but since it's not my codebase, it's totally up to you. Would you like to change type to `SynchronousStreamController`?

Reply:
Lets keep as it is, thanks :)